### ISSUE TITLE
Redox will not boot(Virtualbox, Mac OSX)

### ISSUE THREAD

**Issue by iPyth-on** [redox-os/redox#598]
I just tried to boot the virtual box .vdi file from the website, when I start the machine this is all I get:
![screen shot 2016-03-29 at 1 48 21 pm](https://cloud.githubusercontent.com/assets/13714106/14123420/42b02abe-f5b5-11e5-94dc-1a87a72242a4.png) .

**Comment by ca1ek** [comment]
You might need to switch HDD mode in vbox from IDE to AHCI.

You can do that in the vm settings, in the same place where you mount .iso files. If you have problems with that, reply.

**Comment by ketralnis** [comment]
This isn't my issue but I had the same symptom and can confirm the fix of switching from IDE to AHCI storage controllers

**Comment by leeoniya** [comment]
oh good, had this boot loop too.will test tonight.

**Comment by hughrawlinson** [comment]
I still have this issue. I'm using an ACHI storage controller though, and it's still not working.

**Comment by jdloft** [comment]
Has anything changed from the first screenshot when you guys changed the storage to SCSI AHCI? If so, post a new screenshot.

**Comment by haellowyyn** [comment]
I just tested this on OS X 10.11.6 with VirtualBox 5.1.2. I created a new VM with all settings default and specified the downloaded .vdi as hard disk. The output is slightly different from the above screenshot:

![redox-vbox-osx-3](https://cloud.githubusercontent.com/assets/4521314/17414901/ac958370-5a87-11e6-8791-f8901f7d3321.png)

However, I got the boot to work by replacing the IDE storage controller with a SATA AHCI one. I also had to replace the default PCnet network adapter with an Intel one, or else the DHCP client would panic.

Even with AHCI, VirtualBox still warns me, though. Screenshot, in case this is something that should be fixed:

![redox-vbox-osx](https://cloud.githubusercontent.com/assets/4521314/17415061/667a2eb2-5a88-11e6-9427-a7e15a560ba8.png)

The same happens when building from source. After dismissing the Popup warning and answering 'y' in the console, the boot finishes successfully.

For reference, this is my working VM configuration:

![redox-vbox-osx-4](https://cloud.githubusercontent.com/assets/4521314/17415146/cc451072-5a88-11e6-9e57-385c5192235e.png)
